tensorboard/util.py

```diff
@@ -106,6 +107,7 @@ class Ansi(object):
   """ANSI terminal codes container."""
 
   ESCAPE = '\x1b['
+  ESCAPE_PATTERN = re.compile(re.escape(ESCAPE) + r'\??(\d+)(;\d+)*[mlh]')
```

Comment:
It looks like the first capturing group can be removed and the second can be made non-capturing?